sdk/search/azure-search-documents/README.md

```diff
@@ -210,6 +478,8 @@ do this once across all repos using our CLA.
 This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] 
 or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%2Fsearch%2Fazure-search-documents%2FREADME.png)
```

Comment:
This is a duplicate of line 507. https://github.com/Azure/azure-sdk-for-java/pull/12610/files#diff-5a9f18c5455b82f6c43ef589b69b48ddR507

Reply:
I guess the line number got updated for the latest commits. I did not see the duplication.

Reply:
Looks like it is line 513 now, https://github.com/Azure/azure-sdk-for-java/pull/12610/files#diff-5a9f18c5455b82f6c43ef589b69b48ddR513

Reply:
Found it.